src/components/ui-address-input/ui-address-input.tsx

```diff
@@ -0,0 +1,98 @@
+import { Build, Component, Event, EventEmitter, h, Prop } from "@stencil/core";
+
+const GMAPS_COMPONENT_MAPPING: { [key: string]: string } = {
+  sublocality: "city",
+  locality: "city",
+  postal_code: "zip",
+  route: "street",
+  street_number: "num",
+  administrative_area_level_1: "state",
+};
+
+const toFullAddress = (addressComponents: Array<{ short_name: string; types: Array<string> }>) => {
```

Comment:
@nexussays updated to match how pizzabase format's google addresses 

https://github.com/pizza-to-the-polls/pizzabase/blob/4f44dcec6e34a194cca8a96f15f00e4e3b2cd915/src/lib/validator/geocode.ts#L29-L60